packages/terra-date-picker/src/DatePositionUtil.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Calculates the bottom position of the element.
```

Comment:
This is based on [logic we use for positioning arrows in terra-popup](https://github.com/cerner/terra-framework/blob/master/packages/terra-popup/src/_PopupUtils.js#L187).